.yamato/config/__shared.metafile

```diff
@@ -356,7 +356,7 @@ platforms:
           - command: curl -s https://artifactory.prd.it.unity3d.com/artifactory/unity-tools-local/utr-standalone/utr --output utr
             retries: 2
           - chmod +x utr
-          - command: scp -i ~/.ssh/id_rsa_macmini -r $YAMATO_SOURCE_DIR bokken@$BOKKEN_DEVICE_IP:~/Graphics
```

Comment:
So it's implied that rsync will copy everything over inside a Graphics folder, on the destination? in other words, no need to explicitly have `~/Graphics` anymore?
I guess the test results will confirm :)

Reply:
resolving :: [slack](https://unity.slack.com/archives/D02HZ4S7STT/p1638194181002700) 🙂